### PR TITLE
Adds Various Icons

### PR DESCRIPTION
**Pull request by kurisubrooks**
Added File-Text (#23)
Added Code (#24)
Added Gender/Identity/Sexuality (#27)
Added Language (#36)
Added Half and Full Stars (#48)
Added Package Lock (for NPM 5+)

### CODE REVIEW

**Comment by kurisubrooks** [comment]
Not sure how to add a new category to the site, so identity icons currently don't show.
https://deploy-preview-46--feather.netlify.com/

File: icons/core/code.svg [hunk 1]
```diff
@@ -0,0 +1,4 @@
+<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24">
```
Comment (alexantr) [inline icons/core/code.svg]: `width` and `height` attributes are missed

**Comment by alexantr** [comment]
@kurisubrooks add category to `_includes/icons.html`

**Comment by kurisubrooks** [comment]
Perfect! Thanks @alexantr 
image

File: icons/core/file-text.svg [hunk 2]
```diff
@@ -0,0 +1,6 @@
+<svg xmlns="http://www.w3.org/2000/svg" width="24" height="24" viewBox="0 0 24 24">
+    <path d="M13 2H6a2 2 0 0 0-2 2v16a2 2 0 0 0 2 2h12a2 2 0 0 0 2-2V9z" fill="none" stroke="#000" stroke-linecap="round" stroke-linejoin="round" stroke-width="2"/>
+    <polyline points="13 2 13 9 20 9" fill="none" stroke="#000" stroke-linecap="round" stroke-linejoin="round" stroke-width="2"/>
+    <line x1="16.2" y1="17.87" x2="7.8" y2="17.87" fill="none" stroke="#000" stroke-linecap="round" stroke-linejoin="round" stroke-width="2"/>
```
Comment (alexantr) [inline icons/core/file-text.svg]: Rounded coordinates will be nicely
```
<line x1="16" y1="18" x2="8" y2="18" .../>
<line x1="16" y1="14" x2="8" y2="14" .../>
```

Comment (kurisubrooks) [inline icons/core/file-text.svg]: Oh sure. I just align items to the absolute center of the artboard once i'm done, but i'll update this if it doesn't look too bad.

Comment (alexantr) [inline icons/core/file-text.svg]: Original:
text-1
Rounded:
text-2

Comment (kurisubrooks) [inline icons/core/file-text.svg]: Yeah wow, never realised what a distance it made! Stunning....

**Comment by colebemis** [comment]
Thanks for doing this @kurisubrooks! I'm going to wait until I finish the contribution instructions before I merge anything.

**Comment by colebemis** [comment]
@kurisubrooks How did you find that link to the Netlify deploy preview?

**Comment by andrewda** [comment]
@colebemis It's linked to the Netlify GitHub statuses.

**Comment by colebemis** [comment]
Oh, wow. That's cool!

**Comment by locness3** [comment]
#171 @kurisubrooks @andrewda

**Comment by mittalyashu** [comment]
@kurisubrooks Can you remove the code, file and gender nonbinary icon from the PR? There is already one in feather icons

| Code | File | Gender Nonbinary |
| --- | --- | --- |
| image | image | image |

## Gender female icon

| Icon | Suggestion |
| --- | --- |
| Figma_UbWqyyM3cf | image |

## Gender male icon

Can you increase the size of the icon?

Figma_CPwvDC4rdE

All these icons need to have a minimum of 2px of spacing around the icon

| Gender transgender | Sexuality gay | Sexuality lesbian | Sexuality straight |
| --- | --- | --- | --- |
| image | image | image | image |

**Comment by locness3** [comment]
> All these icons need to have a minimum of 2px of spacing around the icon

No. It can be broken case-by-case as say #171, and it's 1px at least for around the icon.

**Comment by mittalyashu** [comment]
> it's 1px at least for around the icon.

Yes, all 4 icons which I have mentioned inside the table have different spacing.

**Comment by locness3** [comment]
They only need at least 1px.

**Comment by locness3** [comment]
Why did you close, @kurisubrooks ?

**Comment by kurisubrooks** [comment]
Made this a long time ago, and have no interest in updating it. If you'd like, please feel free to take the contents of this PR, fix it up as per the suggestions and resubmit :)

**Comment by locness3** [comment]
Can we also add them to @featherity as well? :)

@jletey

**Comment by johnletey** [comment]
@locness3 Haha I'm `@johnletey` now. We should definitely look into it!

@ericfennis ;)

**Pull request closed without merging**